test/checkPages_test.js

```diff
@@ -50,7 +50,7 @@ function runTest(options, callback) {
 function testOutput(test, log, error, exception) {
   return function(err, context, count) {
     if (err || exception) {
-      test.equal(err.message, exception, 'Wrong exception text');
```

Comment:
I'm afraid I don't remember. I was getting an error (something like "undefined has no message property"), and it wasn't clear to me whether the error was coming from this line or from `checkPages.js`, so I decided to fix it here to reduce confusion for future me.